README.md

```diff
@@ -1,5 +1,8 @@
 # SpO₂
-SpO₂ is oxygen saturation and is used in medical person monitoring.
+
+SpO₂ is a monitor for Kubernetes pods. For more detail and background, see the [blog post announcing the release of the project](https://blog.meilisearch.com/spo2-the-little-dynamic-monitoring-tool/). 
```

Comment:
```suggestion
SpO₂ is a monitor for any program using the HTTP protocol. For more detail and background, see the [blog post announcing the release of the project](https://blog.meilisearch.com/spo2-the-little-dynamic-monitoring-tool/). 
```

Reply:
Sure, that's  a great improvement. I had wanted to make that sort of additional clarification but wasn't positive about the specifics, having only found it today. But sure, go with that, and happy to have helped.

Reply:
BTW, I generally only make PRs for docs, and usually they are just accepted. In this situation, should I do anything more than my last comment? :-)

Reply:
Nothing more is needed, thank you very much again!